src/components/portrait/portrait.spec.js

```diff
@@ -160,6 +170,32 @@ describe("PortraitComponent", () => {
       );
     });
 
+    it("applies expected styles to Portrait with initials on Safari", () => {
+      browserTypeCheck.mockImplementation(() => false);
+      const wrapper = mount(
+        <Portrait shape="circle" initials="AB" darkBackground={false} />
+      );
+
+      assertStyleMatch(
+        {
+          display: "inline-block",
+        },
+        wrapper.find(StyledPortraitContainer)
+      );
+
+      assertStyleMatch(
+        {
+          width: "inherit",
+          height: "inherit",
+          margin: "1px",
+          display: "inline-block",
+          verticalAlign: "middle",
+          boxSizing: "border-box",
```

Comment:
We should be checking for the new `border` style you added for Safari here

Reply:
As per the discussion. Turns out it cannot be tested due to an existing open issue. I'm commented it as a `TODO` in place of this test.